packages/grid_client/src/modules/tft.ts

```diff
@@ -0,0 +1,53 @@
+import { QueryClient } from "@threefold/tfchain_client";
+import Decimal from "decimal.js";
+
+import { expose, validateInput } from "../helpers";
+import { calculator as Calculator } from "./calculator";
+import { TFTModel, USDModel } from "./models";
+
+enum substrateURL {
+  DEV = "wss://tfchain.dev.grid.tf/ws",
+  QA = "wss://tfchain.qa.grid.tf/ws",
+  TEST = "wss://tfchain.test.grid.tf/ws",
+  MAIN = "wss://tfchain.grid.tf/ws",
+}
+
+class TFT {
+  public SUBSTRATE_URL: string;
+  private calculator: Calculator;
+
```

Comment:
add field to represent number of decimals to be passed to the toFixed

Reply:
Added.